container.go

```diff
@@ -103,6 +103,27 @@ type ProcessListItem struct {
 	UserTime100ns                uint64    `json:",omitempty"`
 }
 
+// Type of Request Support in ModifySystem
+type RequestType string
+
+// Type of Resource Support in ModifySystem
+type ResourceType string
+
+// RequestType const
```

Comment:
Am not sure if I understand you question here.
I have defined constants using RequestType and ResourceType. Am defining new type here.